src/harness/testcases/WINNF_FT_S_FDB_testcase.py

```diff
@@ -282,51 +282,68 @@ def generate_FDB_2_default_config(self, filename):
+    # Set the grant frequency to overlap with the DPA 'BATH' which is 3500-3650 MHz.
     grant_g1_a['operationParam']['operationFrequencyRange'] = {
-        'lowFrequency': 3650000000,
-        'highFrequency': 3660000000
+        'lowFrequency': 3640000000,
+        'highFrequency': 3650000000
     }
     grant_g2_a = json.load(
                 open(os.path.join('testcases', 'testdata', 'grant_0.json')))
-    # Set the grant frequency to overlap with the DPA 'puerto_rico_ver2_dpa_4'.
+    # Set the grant frequency to overlap with the DPA 'BATH' which is 3500-3650 MHz.
     grant_g2_a['operationParam']['operationFrequencyRange'] = {
-        'lowFrequency': 3675000000,
-        'highFrequency': 3685000000
+        'lowFrequency': 3615000000,
+        'highFrequency': 3625000000
     }
     grant_g3_a = json.load(
                 open(os.path.join('testcases', 'testdata', 'grant_0.json')))
-    # Set the grant frequency to overlap with the modified DPA 'puerto_rico_ver2_dpa_4'.
+    # Set the grant frequency to overlap with the modified DPA 'BATH' which is 3500-3700 MHz.
     grant_g3_a['operationParam']['operationFrequencyRange'] = {
         'lowFrequency': 3575000000,
         'highFrequency': 3585000000
     }
 
     grant_g1_b = json.load(
                 open(os.path.join('testcases', 'testdata', 'grant_0.json')))
-    # Set the grant frequency to overlap with the DPA 'alaska_dpa_37'.
+    # Set the grant frequency to overlap with the DPA 'China Lake' which is 3500-3650 MHz.
     grant_g1_b['operationParam']['operationFrequencyRange'] = {
-        'lowFrequency': 3650000000,
-        'highFrequency': 3660000000
+        'lowFrequency': 3630000000,
+        'highFrequency': 3640000000
     }
     grant_g2_b = json.load(
                 open(os.path.join('testcases', 'testdata', 'grant_0.json')))
-    # Set the grant frequency to overlap with the DPA 'alaska_dpa_37'.
+    # Set the grant frequency to overlap with the DPA 'China Lake' which is 3500-3650 MHz.
     grant_g2_b['operationParam']['operationFrequencyRange'] = {
-        'lowFrequency': 3675000000,
-        'highFrequency': 3685000000
+        'lowFrequency': 3645000000,
+        'highFrequency': 3655000000
     }
     grant_g3_b = json.load(
                 open(os.path.join('testcases', 'testdata', 'grant_0.json')))
-    # Set the grant frequency to overlap with the modified DPA 'alaska_dpa_37'.
+    # Set the grant frequency to overlap with the modified DPA 'China Lake' which is 3500-3700 MHz.
     grant_g3_b['operationParam']['operationFrequencyRange'] = {
         'lowFrequency': 3575000000,
         'highFrequency': 3585000000
     }
 
+    # Update the location 'X' of CBSD devices to be near DPAs
+    # 'BATH'
+    device_a['installationParam']['latitude'] = 43.906455
+    device_a['installationParam']['longitude'] = -69.813888
+    # 'China Lake'
+    device_b['installationParam']['latitude'] = 37.288360
```

Comment:
This appears to be slightly inside the DPA. Please move to outside.

The device_a location looks good.

Reply:
FDB.2        Updated as per the comment.